libs/net/dal/Services/Interfaces/IContentService.cs

```diff
@@ -8,5 +8,6 @@ namespace TNO.DAL.Services;
 public interface IContentService : IBaseService<Content, long>
 {
     IPaged<Content> Find(ContentFilter filter, bool asNoTracking = true);
+    Task<IPaged<Content>> FindAsync(ContentFilter filter);
```

Comment:
Name needs to indicate source `FindWithElasticsearchAsync`

Reply:
Updated, thanks